MonoidalCategories/gap/BraidedMonoidalCategoriesDerivedMethods.gi

```diff
@@ -6,24 +6,22 @@
 
 ##
 AddDerivationToCAP( BraidingInverseWithGivenTensorProducts,
-                    "BraidingInverseWithGivenTensorProducts as the inverse of the braiding",
-                    [ [ InverseForMorphisms, 1 ],
-                      [ Braiding, 1 ] ],
+                    "BraidingInverseWithGivenTensorProducts via BraidingWithGivenTensorProducts with reordered arguments",
+                    [ [ BraidingWithGivenTensorProducts, 1 ] ],
                     
   function( cat, object_2_tensored_object_1, object_1, object_2, object_1_tensored_object_2 )
-    ##TODO: Use BraidingWithGiven
-    return InverseForMorphisms( cat, Braiding( cat, object_1, object_2 ) );
+    
+    return BraidingWithGivenTensorProducts( cat, object_2_tensored_object_1, object_2, object_1, object_1_tensored_object_2 );
```

Comment:
This only holds in symmetric categories, and already exists in `SymmetricMonoidalCategoriesDerivedMethods.gi`.